apps/www/src/content/blog/import-from-obsidian.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Importing Content From Obsidian
+featuredImageUrl: https://static.feynote.com/screenshots/blog-import-obsidian-20251203.jpg
```

Comment:
This guy isn't a screenshot -- can we put it in either "assets" or in a new "blog-assets" directory?